From 85ea8cf0c54606d68204f508d568c0981a6b87b9 Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Fri, 11 Nov 2011 01:47:19 +0800
Subject: [PATCH 241/505] ENGR00161892 vpu: change MESearchRange parameter for
 encoder in unit test

Change MESearchRange parameter for encoder in unit test. Always 3 for h263
encoder, others can select 0/1/2/3.

Signed-off-by: Sammy He <r62914@freescale.com>
---
 test/mxc_vpu_test/enc.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/test/mxc_vpu_test/enc.c b/test/mxc_vpu_test/enc.c
index 85af3ba..134e41a 100644
--- a/test/mxc_vpu_test/enc.c
+++ b/test/mxc_vpu_test/enc.c
@@ -844,7 +844,8 @@ encoder_open(struct encode *enc)
 
 	encop.IntraCostWeight = 0;
 	encop.MEUseZeroPmv  = 0;
-	encop.MESearchRange = 0;
+	/* (3: 16x16, 2:32x16, 1:64x32, 0:128x64, H.263(Short Header : always 3) */
+	encop.MESearchRange = 3;
 
 	encop.userGamma = (Uint32)(0.75*32768);         /*  (0*32768 <= gamma <= 1*32768) */
 	encop.RcIntervalMode= 1;        /* 0:normal, 1:frame_level, 2:slice_level, 3: user defined Mb_level */
-- 
1.8.0

